<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>620</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>950</width>
    <height>620</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>LAB_Tool</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QTabWidget" name="tab_widget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>950</width>
     <height>600</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Camera</string>
    </attribute>
    <widget class="QLabel" name="label_resolution">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>10</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="text">
      <string>400x300</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="lab_value_change_widget" native="true">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>380</y>
       <width>385</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_12 {border:1px solid #242424;
border-color: rgb(200, 200, 200);}</string>
     </property>
     <widget class="QLabel" name="label_l">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>30</y>
        <width>15</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>L</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_a">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>80</y>
        <width>15</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>A</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_b">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>133</y>
        <width>15</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>B</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="horizontal_slider_l_min">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>35</y>
        <width>169</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontal_slider_a_min">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>85</y>
        <width>169</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontal_slider_b_min">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>135</y>
        <width>169</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_l_min">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>10</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_a_min">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_b_min">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>110</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSlider" name="horizontal_slider_l_max">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>35</y>
        <width>169</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontal_slider_a_max">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>85</y>
        <width>169</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QSlider" name="horizontal_slider_b_max">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>135</y>
        <width>169</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>255</number>
      </property>
      <property name="pageStep">
       <number>1</number>
      </property>
      <property name="value">
       <number>255</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_l_max">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>255</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_a_max">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>255</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_b_max">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>110</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>255</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="lab_manager_widget" native="true">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>380</y>
       <width>85</width>
       <height>161</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#widget_15 {border:1px solid #242424;
border-color: rgb(200, 200, 200);}</string>
     </property>
     <widget class="QComboBox" name="combo_box_color_select">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>10</y>
        <width>69</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="push_button_save_lab">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>120</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{background-color: rgb(255, 165, 0);}
QPushButton:hover{background-color:  rgb(255, 210, 0);}
QPushButton{border-radius:5px;}</string>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_button_add_color">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>40</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{background-color: rgb(255, 165, 0);}
QPushButton:hover{background-color:  rgb(255, 210, 0);}
QPushButton{border-radius:5px;}</string>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
     <widget class="QPushButton" name="push_button_delete_color">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>80</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>黑体</family>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton{background-color: rgb(255, 165, 0);}
QPushButton:hover{background-color:  rgb(255, 210, 0);}
QPushButton{border-radius:5px;}</string>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="push_button_connect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color: rgb(255, 165, 0);}
QPushButton:hover{background-color:  rgb(255, 210, 0);}
QPushButton{border-radius:5px;}</string>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_process">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>50</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_orign">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>50</y>
       <width>400</width>
       <height>300</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="push_button_disconnect">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>480</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color: rgb(255, 165, 0);}
QPushButton:hover{background-color:  rgb(255, 210, 0);}
QPushButton{border-radius:5px;}</string>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontal_slider_servo1">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>400</y>
       <width>181</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>500</number>
     </property>
     <property name="maximum">
      <number>2500</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontal_slider_servo2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>460</y>
       <width>181</width>
       <height>26</height>
      </rect>
     </property>
     <property name="minimum">
      <number>500</number>
     </property>
     <property name="maximum">
      <number>2500</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>1500</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_servo1">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>400</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>servo1</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_servo2">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>460</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>servo2</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_servo1_value">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>380</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>1500</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_servo2_value">
     <property name="geometry">
      <rect>
       <x>800</x>
       <y>440</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="midLineWidth">
      <number>1</number>
     </property>
     <property name="text">
      <string notr="true">1500</string>
     </property>
     <property name="textFormat">
      <enum>Qt::RichText</enum>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="push_button_save_servo">
     <property name="geometry">
      <rect>
       <x>760</x>
       <y>500</y>
       <width>100</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>黑体</family>
       <pointsize>16</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{background-color: rgb(255, 165, 0);}
QPushButton:hover{background-color:  rgb(255, 210, 0);}
QPushButton{border-radius:5px;}</string>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <zorder>lab_manager_widget</zorder>
    <zorder>lab_value_change_widget</zorder>
    <zorder>label_resolution</zorder>
    <zorder>push_button_connect</zorder>
    <zorder>label_process</zorder>
    <zorder>label_orign</zorder>
    <zorder>push_button_disconnect</zorder>
    <zorder>horizontal_slider_servo1</zorder>
    <zorder>horizontal_slider_servo2</zorder>
    <zorder>label_servo1</zorder>
    <zorder>label_servo2</zorder>
    <zorder>label_servo1_value</zorder>
    <zorder>label_servo2_value</zorder>
    <zorder>push_button_save_servo</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
